website/content/docs/install/index.mdx

```diff
@@ -21,32 +21,6 @@ available SHA-256 sums. After downloading Nomad, unzip the package. Make sure
 that the `nomad` binary is available on your `PATH` before continuing with the
 other guides.
 
-You can check the locations available on your path by running this command.
-
-```shell-session
-$ echo $PATH
-
-/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
-```
-
-The output is a list of locations separated by colons. You can make Nomad
-available by moving the binary to one of the listed locations, or by adding
-Nomad's location to your `PATH`.
-
--> **Tip (Linux-based or Mac):** Permanently add a new location to your path by
-editing your shell's settings file (usually called something like `~/.bashrc`,
-where the part of the file name after the `.` and before `rc` is the name of
-your shell). In that file you will see a line that starts with `export PATH=`,
-followed by a colon-separated list of locations. Add the location of the Nomad
-binary to that list and save the file. Then reload your shell's configuration
-with the command `source ~/.bashrc`, replacing `bash` with the name of your
-shell.
-
--> **Tip (Windows):** Add a location to your path via the GUI by navigating to
-`Environment Variables` in your system settings, and looking for the variable
-called `PATH`. You will see a semicolon-separated list of locations. Add the
-Nomad binary's location to that list and then launch a new console window.
-
 </Tab>
 <Tab heading="Linux Packages">
```

Comment:
```suggestion
<Tab heading="Linux Packages" group="linux">
```

Having the matching `group` here keeps the tabs in sync with the ones below. There's also `manual`, `homebrew`, and `chocolatey`.

Also, for some reason the tab name and order here is different from the the Tutorial, so it would be nice for them to match.

<img width="698" alt="image" src="https://user-images.githubusercontent.com/775380/215125661-8e701a11-e811-4638-82d6-47ee93f0f7ea.png">

![image](https://user-images.githubusercontent.com/775380/215125825-55e67bc5-9870-42e0-9980-8e76ec107c7d.png)

Reply:
Oh, cool. Done in 6640357